src/plugins/newsfeed/public/components/newsfeed_header_nav_button.tsx

```diff
@@ -22,11 +24,13 @@ export const NewsfeedContext = React.createContext({} as INewsfeedContext);
 
 export interface Props {
   newsfeedApi: NewsfeedApi;
+  hasCustomBranding$: Observable<boolean>;
 }
 
-export const NewsfeedNavButton = ({ newsfeedApi }: Props) => {
+export const NewsfeedNavButton = ({ newsfeedApi, hasCustomBranding$ }: Props) => {
   const [flyoutVisible, setFlyoutVisible] = useState<boolean>(false);
   const [newsFetchResult, setNewsFetchResult] = useState<FetchResult | null | void>(null);
+  const hasCustomBranding = useObservable(hasCustomBranding$) ?? false;
```

Comment:
`hasCustomBranding$` is not optional. Second parameter of `useObservable` is the default/initial value => `useObservable(hasCustomBranding$, false)`